2022/05/16 22:17:19 config: {myDmux [{sample_name false kafka_http map[dmux:map[batch_size:3 distributor_type:RoundRobin size:4] pending_acks:1e+06 sink:map[breaker_timeout:10s endpoint:http://localhost:8888/1.0/api/event failure_rate_th:0.5 headers:[map[name:X-Client value:go-dumx] map[name:Content-Type value:application/json]] method:POST retry_factor:0.5 retry_interval:100ms timeout:10s window_size:10] source:map[force_restart:true kafka_version_major:2 name:source_name read_newest:true topic:test-four zk_path:localhost:2181/kafka]]}] {file map[enable_debug:true path:default.log rotation:map[compress:true retention_count:5 retention_days:90 size_in_mb:256]]}}
2022/05/16 22:17:19 Starting  kafka_http
2022/05/16 22:17:19 Connected to 127.0.0.1:2181
2022/05/16 22:17:19 Authenticated: id=72078516766244959, timeout=6000
2022/05/16 22:17:19 Re-submitting `0` credentials after reconnect
2022/05/16 22:19:42 /test-four/0//34389 after kafka source
2022/05/16 22:19:50 Recv loop terminated: err=read tcp 127.0.0.1:57056->127.0.0.1:2181: i/o timeout
2022/05/16 22:19:54 Send loop terminated: err=write tcp 127.0.0.1:57056->127.0.0.1:2181: write: broken pipe
2022/05/16 22:19:54 Connected to [::1]:2181
2022/05/16 22:19:54 Authentication failed: zk: session has been expired by the server
2022/05/16 22:19:55 Connected to 127.0.0.1:2181
2022/05/16 22:19:55 Authenticated: id=72078516766244960, timeout=6000
2022/05/16 22:19:55 Re-submitting `0` credentials after reconnect
2022/05/16 22:20:11 /test-four/1//29834 after kafka source
2022/05/16 22:20:30 Send loop terminated: err=write tcp 127.0.0.1:57121->127.0.0.1:2181: write: broken pipe
2022/05/16 22:20:47 Recv loop terminated: err=read tcp 127.0.0.1:57121->127.0.0.1:2181: read: connection reset by peer
2022/05/16 22:20:50 Connected to [::1]:2181
2022/05/16 22:20:50 Authentication failed: zk: session has been expired by the server
2022/05/16 22:20:51 Connected to 127.0.0.1:2181
2022/05/16 22:20:51 Authenticated: id=72078516766244961, timeout=6000
2022/05/16 22:20:51 Re-submitting `0` credentials after reconnect
2022/05/16 22:21:25 config: {myDmux [{sample_name false kafka_http map[dmux:map[batch_size:3 distributor_type:RoundRobin size:4] pending_acks:1e+06 sink:map[breaker_timeout:10s endpoint:http://localhost:8888/1.0/api/event failure_rate_th:0.5 headers:[map[name:X-Client value:go-dumx] map[name:Content-Type value:application/json]] method:POST retry_factor:0.5 retry_interval:100ms timeout:10s window_size:10] source:map[force_restart:true kafka_version_major:2 name:source_name read_newest:true topic:test-four zk_path:localhost:2181/kafka]]}] {file map[enable_debug:true path:default.log rotation:map[compress:true retention_count:5 retention_days:90 size_in_mb:256]]}}
2022/05/16 22:21:25 Starting  kafka_http
2022/05/16 22:21:29 Connected to [::1]:2181
2022/05/16 22:21:29 Authenticated: id=72078516766244962, timeout=6000
2022/05/16 22:21:29 Re-submitting `0` credentials after reconnect
2022/05/16 22:22:02 /test-four/1//29835 after kafka source
2022/05/16 22:22:07 Recv loop terminated: err=read tcp [::1]:57170->[::1]:2181: i/o timeout
2022/05/16 22:22:07 Send loop terminated: err=<nil>
2022/05/16 22:22:07 Connected to 127.0.0.1:2181
2022/05/16 22:22:07 Authenticated: id=72078516766244962, timeout=6000
2022/05/16 22:22:07 Re-submitting `0` credentials after reconnect
2022/05/16 22:22:13 /test-four/3//32191 after kafka source
2022/05/16 22:22:13 /test-four/3//32192 after kafka source
2022/05/16 22:23:26 Recv loop terminated: err=EOF
2022/05/16 22:23:26 Send loop terminated: err=<nil>
2022/05/16 22:23:26 /test-four/1//29835 before http sink
2022/05/16 22:23:26 /test-four/3//32191 before http sink
2022/05/16 22:23:26 /test-four/3//32192 before http sink
2022/05/16 22:23:26 Connected to [::1]:2181
2022/05/16 22:23:26 Authentication failed: zk: session has been expired by the server
2022/05/16 22:23:26 failed in http call invoke http://localhost:8888/1.0/api/event/test-four?batch=1,,29835~3,,32191~3,,32192 Post "http://localhost:8888/1.0/api/event/test-four?batch=1,,29835~3,,32191~3,,32192": dial tcp [::1]:8888: connect: connection refused 
2022/05/16 22:23:26 worker.goroutine: sending signal 3
2022/05/16 22:23:26 retry in execute POST 	 http://localhost:8888/1.0/api/event/test-four?batch=1,,29835~3,,32191~3,,32192
2022/05/16 22:23:27 Connected to 127.0.0.1:2181
2022/05/16 22:23:27 Authenticated: id=72078516766244963, timeout=6000
2022/05/16 22:23:27 Re-submitting `0` credentials after reconnect
2022/05/16 22:23:48 Recv loop terminated: err=EOF
2022/05/16 22:23:48 Send loop terminated: err=<nil>
2022/05/16 22:23:48 Failed to connect to [::1]:2181: dial tcp [::1]:2181: connect: connection refused
2022/05/16 22:23:49 Failed to connect to 127.0.0.1:2181: dial tcp 127.0.0.1:2181: connect: connection refused
2022/05/16 22:23:49 Failed to connect to [::1]:2181: dial tcp [::1]:2181: connect: connection refused
2022/05/16 22:23:50 Failed to connect to 127.0.0.1:2181: dial tcp 127.0.0.1:2181: connect: connection refused
2022/05/16 22:23:50 Failed to connect to [::1]:2181: dial tcp [::1]:2181: connect: connection refused
2022/05/16 22:23:51 Connected to 127.0.0.1:2181
2022/05/16 22:23:51 Authentication failed: EOF
2022/05/16 22:23:51 Connected to [::1]:2181
2022/05/16 22:23:51 Authentication failed: EOF
2022/05/16 22:23:52 Connected to 127.0.0.1:2181
2022/05/16 22:23:52 Authenticated: id=72078516766244963, timeout=6000
2022/05/16 22:23:52 Re-submitting `0` credentials after reconnect
